web.py

```diff
@@ -49,7 +55,7 @@
 os.makedirs(os.path.join(now_dir, "assets/weights"), exist_ok=True)
 os.environ["TEMP"] = tmp
 warnings.filterwarnings("ignore")
-torch.manual_seed(114514)
+torch.manual_seed(114514)  # ????
```

Comment:
This is a meme number in the Chinese and Japanese Internet.

Reply:
> This is a meme number in the Chinese and Japanese Internet.

That makes a lot more sense to be honest lol

Reply:
And also a number 23333, if you see some other codes.